modules/swagger-codegen/src/main/resources/php/api.mustache

```diff
@@ -172,6 +201,33 @@ use \{{invokerPackage}}\ObjectSerializer;
 
         {{#formParams}}// form params
         if (${{paramName}} !== null) {
+            {{#hasValidation}}
```

Comment:
Yes, you are right. 
All kind of params(queryParams, formParams.. ) should be validated in the allParams tag. I'll fix it